2018-westphal-antpc.tex

```diff
@@ -1,14 +1,17 @@
 \documentclass{anstrans}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\title{Signatures and Observables of the Nuclear Fuel Cycle}
-\author{\textbf{Gregory T. Westphal}, and Kathryn D. Huff}
+\title{PyRe: A Cyclus Pyroprocessing Repository}
```

Comment:
Repository is an overloaded term in nuclear. What you're creating isn't a repository (ok, yes, it is in a git repository... but that's beside the point). It's a facility 'archetype'. 

Anyway, it doesn't need to be an acroynym. 

'Pyre: A Cyclus Pyroprocessing Facility Archetype'